src/pages/SupplierInvoicePage.js

```diff
@@ -131,7 +131,11 @@ export const SupplierInvoice = ({
       <View style={pageTopSectionContainer}>
         <View style={pageTopLeftSectionContainer}>
           <PageInfo columns={pageInfoColumns} isEditingDisabled={isFinalised} />
-          <SearchBar onChangeText={onFilterData} value={searchTerm} />
+          <SearchBar
+            onChangeText={onFilterData}
+            value={searchTerm}
+            placeholder={`${generalStrings.searchBar} ${tableStrings.name}`}
```

Comment:
Do the values for the column show for you?

![image](https://user-images.githubusercontent.com/7684221/71867658-ecab4680-316f-11ea-9e25-8e9e5dc35a2c.png)